.eslintrc.json

```diff
@@ -6,9 +6,8 @@
 		"import/no-extraneous-dependencies": [
 			"error",
 			{
-				"devDependencies": ["test/**"],
-				"optionalDependencies": false,
-				"packageDir": "./"
+				"devDependencies": true,
```

Comment:
This should be a pattern, not a blanket `true`. It needs to exclude `src` in any case.